### ISSUE TITLE
 How to add '; ' after each style

### ISSUE THREAD

**Issue by takegine** [sass/sass#3517]
in my addon, `outputStyles:expanded` is working ,but error when `outputStyles:compressed`

```
.root {
  width: 100%;
  height: 100%;
}
#XPCircular {
  width: 100%;
  height: 100%;
}
```

or 
```
.root{width:100%;height:100%;}#XPCircular {width: 100%;height: 100%;}
```

those are working 

but 

```
.root{width:100%;height:100%}#XPCircular {width: 100%;height: 100%}
```
this output from `outputStyles:compressed` is not working 

i need  `replace(/}/g, ';}' )`
Have any config to do it ?

**Comment by stof** [comment]
The `;` is optional after the last style rule in CSS. So this is a bug of the tool not supporting the absence of the `;`, which is where you should report it.

**Comment by takegine** [comment]
> The `;` is optional after the last style rule in CSS. So this is a bug of the tool not supporting the absence of the `;`, which is where you should report it.

tool is working by  this function require form sass 
`export function renderSync(options: LegacyOptions<'sync'>): LegacyResult;`

but i have no idea, which optional is useful

**Comment by takegine** [comment]
And i saw the example is without `;` after the last style
![image](https://user-images.githubusercontent.com/64788603/229264179-029af542-5b5c-4fbf-bd62-9ae262814eda.png)

**Comment by stof** [comment]
I don't understand your answer.

The compressed output generated by Sass is perfectly valid CSS, because the `;` after the last property is marked as optional in the CSS spec. So any spec-compliant CSS parser **must** support such CSS.
This means that if something breaks in your project without the semicolon, it means you have a broken CSS parser somewhere in your stack, which is what should be fixed instead of asking Sass to generate a less compressed output for the compressed style.
